lib/elix/responders/lists.ex

```diff
@@ -11,7 +11,7 @@ defmodule Elix.Responders.Lists do
   show lists - Displays all lists
   """
   respond ~r/show lists\Z/i, msg do
-    reply(msg, render_items(List.all_names))
+    send(msg, render_items(List.all_names))
```

Comment:
Should `List.allnames` return a nullobject-ey thing so you don't need to change this logic?